keps/sig-storage/2485-read-write-once-pod-pv-access-mode/README.md

```diff
@@ -832,13 +948,31 @@ Try to be as paranoid as possible - e.g., what if some components will restart
 mid-rollout?
 -->
 
+Rolling out this feature involves enabling the ReadWriteOncePod feature gate
+across kube-apiserver, kube-scheduler, kubelet, and updating CSI driver and
+sidecar versions.
```

Comment:
Does the sidecar need to be upgraded before enabling flags and allowing clients to use the feature? Reading the [version skew section about the sidecar](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2485-read-write-once-pod-pv-access-mode) I was unclear on this part.

Reply:
Updated to clarify that the order of operations does not matter.